tests/docker/config/search/construct-query.sparql

```diff
@@ -242,6 +242,8 @@ CONSTRUCT {
   ?alias :singleNeuronSimulation  ?singleNeuronSimulation  .
                                   ?singleNeuronSimulation  :injectionLocation                ?injectionLocation .
                                   ?singleNeuronSimulation  :recordingLocation                ?recordingLocation .
+                                  ?singleNeuronSimulation  :isDraft                          ?isDraft .
+                                  ?singleNeuronSimulation  :status                           ?status  .
```

Comment:
`status` is already used by simulation campaigns, are we sure they are not clashing each other ?

Reply:
are they indexed together? how is the batch constructed?

Reply:
is there a way to test this?

Reply:
They can as the batch takes the state log in its chronological order

Reply:
then isn't it the same for `Simulation` VS `SimulationCampaign`, they both use status